decidim-initiatives/spec/mailers/decidim/initiatives/initiatives_mailer_spec.rb

```diff
@@ -40,7 +40,7 @@ module Initiatives
           end
 
           it "renders the body" do
-            expect(mail.body.encoded).to match(initiative.title["en"])
+            expect(mail.body.encoded).to include(decidim_escape_translated(initiative.title))
```

Comment:
I know we're not consistent in the same file but I'd prefer to keep the consistency between v0.27/v0.28/develop
```suggestion
            expect(mail.body.encoded).to include(decidim_html_escape(initiative.title))
```